docs/releases/v8_13.md

```diff
@@ -0,0 +1,37 @@
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Amiberry**](../../software/gaming/#amiberry) :octicons-arrow-right-16: Updated to version 5.5.1, including `LibSDL2` v2.26.2. The update can be applied via reinstall: `dietpi-software reinstall 108`
+
+### Bug fixes
+
+- [**NanoPi M2/T2/Fire2**](../../hardware/#nanopi-series-friendlyelec) :octicons-arrow-right-16: Updated our image with a device tree for the actual NanoPi M2 and a fixed device tree to allow boot on NanoPi 2 Fire. Many thanks to @NewbieOrange for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5555>
+- [**PINE H64**](../..//hardware/#pine64) :octicons-arrow-right-16: Resolved an issue where our recent PINE H64 images did not contain a kernel. Many thanks to @DremOSDeveloperTeam for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6031>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**DietPi-LetsEncrypt**](../../software/system_security/#lets-encrypt) :octicons-arrow-right-16: Resolved an error with Nginx when multiple "server_name" directives were present in `/etc/nginx/sites-available/default` due to e.g. a repeating `dietpi-letsencrypt`/Certbot execution or manually added vhosts. Many thanks to @hemertje for reporting this issue: <https://dietpi.com/forum/t/nextcloud-problems-questions/15415/13>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Readarr**](../../software/bittorrent/#readarr) :octicons-arrow-right-16: Resolved an issue where the installation failed because of a wrong download URL. Many thanks to @lambolighting for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5992>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Docker**](../../software/programming/#docker) :octicons-arrow-right-16: Resolved an issue where the install failed if the kernel did not support nftables. Since iptables is installed as dependency along with the Docker package, it cannot be configured first to use the legacy API where needed. The Docker service is now masked before the package is installed, to prevent it from starting during the package install, before we can configure iptables. Many thanks to @tggjifdhn for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6013>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**myMPD**](../../software/media/#mympd) :octicons-arrow-right-16: Resolved an issue where APT updates failed since the myMPD APT repository key expired on 2023-01-01. Many thanks to @cpsjones for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6044>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Homer**](../../software/system_stats/#homer) :octicons-arrow-right-16: Resolved an issue where reinstalls failed. Many thanks to @JappeHallunken for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6060>
```

Comment:
```suggestion
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Homer**](../../software/system_stats/#homer) :octicons-arrow-right-16: Resolved an issue where reinstalls failed. Many thanks to @JappeHallunken for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6060>
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Grafana**](../../software/hardware_projects/#grafana) :octicons-arrow-right-16: Resolved an issue where APT updates failed if Grafana was installed, since the repo key was rotated on January 12: <https://grafana.com/blog/2023/01/12/grafana-labs-update-regarding-circleci-security-updates/>
```